packages/api/.eslintrc.json

```diff
@@ -0,0 +1,7 @@
+{
+    "extends": "@comet/eslint-config/nestjs",
+    "ignorePatterns": ["src/mikro-orm/migrations/**", "lib/**"],
+    "rules": {
+        "@comet/no-other-module-relative-import": "off"
```

Comment:
Why do you need to deactivate this rule?

Reply:
This is not needed here. I removed this in 342f4bf10c40e957ca89d89de2e1877ed50a6938